### PR TITLE
RDISCROWD-4793 Fix announcements permission to clear cache

### PR DESCRIPTION
**Pull request by kbecker42**
- Clear announcements cache when updating an announcement.
- Fixes bug with user-level workers seeing admin-level announcements during cache period of announcement between saves.

### CODE REVIEW

File: pybossa/cache/announcements.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
```
Comment (dchhabda) [inline pybossa/cache/announcements.py:4]: this can be updated to 2021.

**Review by dchhabda — DISMISSED** [review]
lgtm. new copyright comment can be updated to latest year.

File: pybossa/cache/announcements.py (hunk 1 above)
Comment (kbecker42) [reply pybossa/cache/announcements.py]: Fixed via https://github.com/bloomberg/pybossa/commit/08230b6aeb6ff0b5ec32066729dd36e7e0c88cab

**Review by ywrsusan — APPROVED** [review]
lgtm

**Pull request merged**